nextjs/src/app/reports/[id]/(components)/ReportNavbar.tsx

```diff
@@ -132,12 +136,12 @@ export default function ReportNavbar() {
               <TableComboBox />
             )}
             <ReportStarredItemsDropdown />
-            {!!explorer.isValidEntity(explorer.state) && (
-              <PanelRight
-                onClick={explorer.toggle}
-                className="text-meepGray-400 w-4 h-4 cursor-pointer ml-3"
-              />
-            )}
+            {/* {!!explorer.isValidEntity(explorer.state) && ( */}
```

Comment:
question: should this commented code be removed?

Reply:
Nope, see https://github.com/commonknowledge/mapped/pull/206#discussion_r1939698329